_data/conferences.yml

```
@@ -49,6 +49,13 @@
   end: 2020-11-05
   location: 🇰🇷 Seoul, South Korea
   cocoa-only: true
+  
+  - name: FLutter Forward
```

```suggestion
- name: Flutter Forward
```
yml indenting is a bit tricky :)